src/Engines/ElasticSearchEngine.php

```diff
@@ -117,7 +120,8 @@ public function getTotalCount($results)
 
     /**
      * @param BaseBuilder $builder
-     * @param array $options
+     * @param array       $options
```

Comment:
This is unnecessary changes

Reply:
I'll fix this and would "Add HitsIteratorAggregate for custom binding" be a more imperative tone?

Reply:
You right it would be more imperative, or you can use `Replace harcoded EloquentHitsIteratorAggregate to an interface`

Reply:
Done, fixed the above mentioned issue and renamed commit to "Replace harcoded EloquentHitsIteratorAggregate to an interface"